src/commands/QuitCommand.cpp

```diff
@@ -1,16 +1,11 @@
 #include "QuitCommand.hpp"
 
-/**
- * QuitCommand default constructor.
- */
-QuitCommand::QuitCommand() : ACommand(false), _msg(NONE) {}
-
 /**
  * QuitCommand message constructor.
  * 
  * @param msg The message
  */
-QuitCommand::QuitCommand(std::string msg) : ACommand(false), _msg(msg) {}
+QuitCommand::QuitCommand(std::string message) : ACommand(false), _message(message) {}
```

Comment:
aprovechamos para hacer un 
```suggestion
QuitCommand::QuitCommand(const std::string &message) : ACommand(false), _message(message) {}
```
?